bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/i18n/tesla.properties

```diff
@@ -296,3 +292,10 @@ channel-type.tesla.wakeup.label = Wake Up
 channel-type.tesla.wakeup.description = Wake up the vehicle from a (deep) sleep
 channel-type.tesla.wiperbladeheater.label = Wiperblade Heater
 channel-type.tesla.wiperbladeheater.description = Indicates if the wiperblade heater is switched on
+
+# thing types config
+
+thing-type.config.tesla.account.password.label = Password
+thing-type.config.tesla.account.password.description = Password for the Tesla Remote Service
+thing-type.config.tesla.account.username.label = Username
+thing-type.config.tesla.account.username.description = Username for the Tesla Remote Service, e.g email address.
```

Comment:
Is this auto-generated? Seems strange that a new thing types config section would be generated here when there's already a section from line 19.

Reply:
Yes, I purely ran the generator.

Reply:
But you are right, these entries should actually have disappeared as we do no longer have these parameters...

Reply:
I've now deleted it and regenerated - it didn't appear again.

Reply:
Perfect! Thanks.